### PR TITLE
fixed scripts place, and added clarification to the readme

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
fixed scripts place and added clarification to required permissions inside the readme file

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+#### Scripts
+##### MS365DefenderUserListToTable
+- The MS365DefenderUserListToTable folder is now inside the Scripts folder.
+##### MS365DefenderCountIncidentCategories
+- The MS365DefenderCountIncidentCategories folder is now inside the Scripts folder.
```
Comment (roysagi) [inline Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md:6]: These are dev notes - not something visible to the user
just put `Maintenance and stability enhancements.` instead for both of these

Also put a note for the docker image updates

File: Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+#### integration's README.md
+- Added clarification regarding the required permissions.
```
Comment (roysagi) [inline Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md:9]: I don't think this requires a note - did you add this manually?

**Review by roysagi — CHANGES_REQUESTED** [review]

File: Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md (hunk 2 above)
Comment (iyeshaya) [reply Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md]: Removed

File: Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md (hunk 1 above)
Comment (iyeshaya) [reply Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md]: Fixed

File: Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+- Maintenance and stability enhancements.
+##### MS365DefenderUserListToTable
+- Upgraded the Docker image to 3.9.6.22912
```
Comment (roysagi) [inline Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.6.22912*.
```

File: Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+##### MS365DefenderCountIncidentCategories
+- Upgraded the Docker image to 3.9.6.22912
```
Comment (roysagi) [inline Packs/Microsoft365Defender/ReleaseNotes/1_1_2.md:7]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.6.22912*.
```

**Review by roysagi — APPROVED** [review]

**Pull request merged**